locales/fr.json

```diff
@@ -44,6 +44,8 @@
     "backup_output_directory_required": "Vous devez spécifier un dossier de destination pour la sauvegarde",
     "backup_running_hooks": "Exécution des scripts de sauvegarde...",
     "custom_app_url_required": "Vous devez spécifier une URL pour mettre à jour votre application personnalisée {app:s}",
+    "disk_space_not_sufficient_install": "Il ne reste pas assez d'espace disque pour installer cette application",
+    "disk_space_not_sufficient_update": "Il ne reste pas assez d'espace disque pour mettre à jour cette application",
```

Comment:
It should be okay, but note for next time : we tend to avoid editing locale files manually - apart from `en.json`, because this may create merge conflicts on our translation platform weblate